### ISSUE TITLE
Tackling a series of Accessibility issues to improve score on Index Page

### ISSUE THREAD

**Issue by mariamalth** [CMU-17313Q/Mayan-EDMS#8]
The current accessibility score is 89. The problems i will be tackling include a lack of contrast between the background and non-unique ARIA ids.
